### PR TITLE
#252 Add maps/gki-bel-cyrl-latn-2000 & fix id in maps/by-bel-Cyrl-Latn-2007

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #252 
 - #251

### CODE REVIEW

File: maps/by-bel-Cyrl-Latn-2007.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 authority_id: by
-id: 9303
+id: '8/16668'
```
Comment (ronaldtse) [inline maps/by-bel-Cyrl-Latn-2007.yaml]: Can we change this to `id: 2007` to indicate year? Thanks.

File: maps/gki-bel-cyrl-latn-2000.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+authority_id: gki
+id: '8/4488'
```
Comment (ronaldtse) [inline maps/gki-bel-cyrl-latn-2000.yaml]: Can we change this to `id: 2000`?

File: maps/un-bel-Cyrl-Latn-2007.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
+---
+authority_id: un
+id: '8/16668'
```
Comment (ronaldtse) [inline maps/un-bel-Cyrl-Latn-2007.yaml]: Can we change this to `id: 2007`? Thanks.

**Review by ronaldtse — APPROVED** [review]
Thanks @CAMOBAP795 please merge when test passes!

File: maps/by-bel-Cyrl-Latn-2007.yaml (hunk 1 above)
Comment (CAMOBAP) [reply maps/by-bel-Cyrl-Latn-2007.yaml]: Done

File: maps/un-bel-Cyrl-Latn-2007.yaml (hunk 3 above)
Comment (CAMOBAP) [reply maps/un-bel-Cyrl-Latn-2007.yaml]: Done

File: maps/gki-bel-cyrl-latn-2000.yaml (hunk 2 above)
Comment (CAMOBAP) [reply maps/gki-bel-cyrl-latn-2000.yaml]: Done

**Pull request merged**